ex1/ex1.c

```diff
@@ -9,6 +9,19 @@
 int main(void)
```

Comment:
not quite done yet, Tai. you stiill need to set `x` to some value and change that value within the child and parent process.